feature-about/src/main/res/values-en/strings.xml

```diff
@@ -7,4 +7,5 @@
     <string name="about_staff">Staff List</string>
     <string name="about_privacy">Privacy Policy</string>
     <string name="about_license">License</string>
+    <string name="about_code_conduct">Code Conduct</string>
```

Comment:
This should be moved to `core.model/commonMain/resources`

Reply:
Automatically moved by merging main branch 👍🏼